sdk/ml/azure-ai-ml/tests/internal/unittests/test_pipeline_job.py

```diff
@@ -602,3 +606,14 @@ def test_job_properties(self):
             assert "AZURE_ML_PathOnCompute_" in list(node_dict["properties"].keys())[0]
             assert node_dict["properties"] == rest_node_dict["properties"]
 
+    def test_load_pipeline_job_with_internal_nodes_from_rest(self):
+        # this is a simplified test case which avoid constructing a complete pipeline job rest object
+        from azure.ai.ml.entities._job.pipeline._load_component import pipeline_node_factory
+
+        unregister_internal_components()
```

Comment:
why unregister here? Did we opened internal component flag for CI?

Reply:
enabled for this test. Here is to check whether it will **try** register internal components on `PipelineJob._from_rest_object`